runtime/jcl/common/criusupport.c

```diff
@@ -21,11 +21,73 @@
  *******************************************************************************/
 #if defined(LINUX)
 #include <criu/criu.h>
+#include <fcntl.h>
+#include <errno.h>
+
+#include "ut_j9jcl.h"
 #endif /* defined(LINUX) */
 
 #include "jni.h"
 #include "j9.h"
 #include "jclprots.h"
+#include "j9jclnls.h"
+
+#define STRING_BUFFER_SIZE 256
+
+static void
+setupJNIFieldIDs(JNIEnv *env)
+{
+	J9VMThread *currentThread = (J9VMThread*)env;
+	J9JavaVM *vm = currentThread->javaVM;
+	J9InternalVMFunctions *vmFuncs = vm->internalVMFunctions;
+	jclass criuResultTypeClass = NULL;
+	jclass criuResultClass = NULL;
+
+	criuResultTypeClass = (*env)->FindClass(env, "com/ibm/jvm/CRIUSupport$CRIUResultType");
```

Comment:
```suggestion
	criuResultTypeClass = (*env)->FindClass(env, "org/eclipse/openj9/CRIUSupport$CRIUResultType");
```

Sorry, catching this late but I'm wondering why we're creating new APIs in the `com.ibm.*` name space rather than in the `org/eclipse/openj9` namespace?

Reply:
I've made the change